From: David Tardon <dtardon@redhat.com>
Date: Thu, 16 May 2024 08:59:50 +0200
Subject: journald-server: drop invalid assert()

config_parse_compress() is not used for parsing unit files, hence unit
is always NULL.

Fixes #32856

Follow-up-for: 6813be2eba24a2f864d5d47326437dbff38cd675
---
 src/journal/journald-server.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/journal/journald-server.c b/src/journal/journald-server.c
index 1d03350..950ec22 100644
--- a/src/journal/journald-server.c
+++ b/src/journal/journald-server.c
@@ -3012,7 +3012,6 @@ int config_parse_compress(
         JournalCompressOptions* compress = ASSERT_PTR(data);
         int r;
 
-        assert(unit);
         assert(filename);
         assert(rvalue);
 
